meta/run.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# In this file we specify how each microservice should boot.
+#
+
+runtime: native
+
+config_set: 
+   keyvaluestore:
+      bootcmd: /node keyvaluestore.js
+      env:
+         PORT: 9000
+         MICRO_MASTER_ENDPOINT: micro-master.default.svc.cluster.local
+         MICRO_DB_ENDPOINT: micro-db.default.svc.cluster.local
+         MICRO_STORAGE_ENDPOINT: micro-storage.default.svc.cluster.local
+   db:
+      bootcmd: /node db.js
+      env:
+         PORT: 9001
+         MICRO_KEYVALUESTORE_ENDPOINT: micro-keyvaluestore.default.svc.cluster.local
```

Comment:
With the introduction of https://github.com/mikelangelo-project/capstan/pull/22, users will now be able to do something like

```
capstan run micro --boot db --env MICRO_KEYVALUESTORE_ENDPOINT=IP --PORT=8001
```

This will set the command of OSv to

```
--env=MICRO_KEYVALUESTORE_ENDPOINT=IP --env=PORT=8001 --runscript /run/db
```

but `/run/db` will also have corresponding `--env` s set based on the run config.

Thus, which env variables will be used? I think those given to `Capstan` should have preference.